src/com/esotericsoftware/kryo/Kryo.java

```diff
@@ -1187,23 +1200,41 @@ public boolean isClosure (Class type) {
 
 	/** Tracks the generic type arguments and actual classes for type variables in the object graph during seralization.
 	 * <p>
-	 * When serializing a type with a single type parameter, {@link Generics#nextGenericClass() nextGenericClass} will return the
-	 * generic class (or null) and must be followed by {@link Generics#popGenericType() popGenericType}. See
-	 * {@link CollectionSerializer} for an example.
+	 * When serializing a type with a single type parameter, {@link DefaultGenericsStrategy#nextGenericClass() nextGenericClass}

```

Comment:
Should this (and the following) rather refer to `GenericsStrategy` instead of `DefaultGenericsStrategy`? Probably this is a result of a rename ;-)

Reply:
You're right :)